src/components/cards/LevelCard.tsx

```diff
@@ -0,0 +1,29 @@
+import classNames from "classnames";
+import React from "react";
+
+const LevelCard = ({
+  level,
+  title,
+  description,
+}: {
+  level: number;
+  title: string;
+  description: string;
+}) => {
+  const bgColor = classNames({
+    "bg-[#dae8fc] border-[#6c8ebf]": level == 0,
+    "bg-[#d5e8d4] border-[#82b366]": level == 1,
+    "bg-[#ffe6cc] border-[#d79b00]": level == 2,
+    "bg-[#e1d5e7] border-[#9673a6]": level == 3,
+  });
```

Comment:
I do not know if It is necessary to add this color in tailwind.config since we are using them here only. I do not know